src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -108,10 +108,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }))
-
             .OnExecuteCompleted(pr =>
             {
-                notifications.ShowMessage(String.Format(CultureInfo.CurrentCulture, Resources.PRCreatedUpstream, TargetBranch.Id,
+                notifications.ShowMessage(String.Format(CultureInfo.CurrentCulture, Resources.PRCreatedUpstream, TargetBranch.Repository.Owner + "/" + TargetBranch.Repository.Name,
```

Comment:
@shana you mentioned I merge in #652 into here, but remind me again what I was supposed to do here? 😅 The message currently displays the owner and the repo, but not the branch.